sycl/doc/extensions/LevelZeroBackend/LevelZeroBackend.md

```diff
@@ -101,11 +103,11 @@ some interoparability API supports overriding this behavior and keep the ownersh
 Use this enumeration for explicit specification of the ownership:
 ``` C++
 namespace sycl {
-namespace level_zero {
+  namespace level_zero {
 
-enum class ownership { transfer, keep };
+  enum class ownership { transfer, keep };
 
-} // namesace level_zero
+  } // namesace level_zero
 } // namespace sycl

```

Comment:
Why `level_zero` namespace is moved? Probably we should make style changes separately as I'm not sure if they are aligned with the coding guidelines.

Reply:
reverted clang-format changes